meta/state.go

```diff
@@ -62,6 +64,16 @@ func (r *localRaft) remove() error {
 	return nil
 }
 
+func (r *localRaft) shutdown() error {
```

Comment:
I don't see `close` calling this to stop raft?  Pretty sure it's the same logic, no?